cmd/create/cluster/cmd.go

```diff
@@ -1237,7 +1245,7 @@ func run(cmd *cobra.Command, _ []string) {
 		r.Reporter.Errorf("Expected a valid AWS region")
 		os.Exit(1)
 	} else {
-		if isHostedCP && !interactive.Enabled() {
+		if isHostedCP && !interactive.Enabled() && !shardPinningEnabled {
```

Comment:
I think this might be incorrect. Why is this region check being done only when shad pinning is _not_ enabled?